src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -437,16 +430,14 @@ private boolean buildGraphOverWire() {
                         response = scanner.next();
                     }
                     statusMessage = String.format("Got response code %d from server due to %s", code, response);
```

Comment:
Inline `statusMessage`.

Reply:
Also, based on how `statusMessage` is used, it can probably a local variable instead of a field.